cypress/cypress.config.ts

```diff
@@ -29,6 +29,7 @@ module.exports = defineConfig({
   env: {
     auth_type: AuthType.Cognito,
     base_url: "",
+    lambda_endpoint: "",
```

Comment:
Can you add some documentation on this parameter and when you'd use it (running UI locally against deployed API GW for lambdas)?